locales/it/index.yml

```diff
@@ -568,6 +568,8 @@ services:
   contactsAndInfo: Contatti ed informazioni
   contactAddress: Indirizzo
   contactPhone: Telefono
+  search:
+    invalidSearchBarText: Inserisci almeno tre caratteri nella barra di ricerca.
```

Comment:
since this is the same text used for all searches, I suggest to move it to `global.search. invalidSearchBarText` and also change the messages search to use that - cc @fpersico

Reply:
in the PR i fixed that